pxr/imaging/hdSt/materialXFilter.cpp

```diff
@@ -61,31 +61,86 @@ TF_DEFINE_PRIVATE_TOKENS(
     ((defaultInput, "default"))
 
     // Opacity Parameters
+    (UsdPreviewSurface)
     (opacity)
     (opacityThreshold)
+
+    (standard_surface)
     (transmission)
 
+    (open_pbr_surface)
```

Comment:
Added programmatic detection for both OpenPBR and glTf surfaces to be able to get the materialTag without having to peek into the MaterialX NodeGraph implementation.

Reply:
Ahh, just merged https://github.com/PixarAnimationStudios/OpenUSD/pull/3069 ...  Any quick opinions on how to merge those changes vs these?

Reply:
This PR also includes detection for glTf textures via a [_GetGlTFSurfaceMaterialTag()](https://github.com/PixarAnimationStudios/OpenUSD/pull/3073/files#diff-c5e0ffdbaf94bdfaffbc73413cb05018d1a3115640d2c53d51ddff9c886be346R856) function that properly differentiate between translucent and masked mode. The merge resolution strategy would be to let this PR fully win over #3069, equivalent to reverting it before merge.

I have opened Pablo's asset with my local build and it renders correctly.